src/plugins/home/kibana.json

```diff
@@ -8,6 +8,6 @@
   "server": true,
   "ui": true,
   "requiredPlugins": ["dataViews", "share", "urlForwarding"],
-  "optionalPlugins": ["usageCollection", "telemetry", "customIntegrations"],
+  "optionalPlugins": ["usageCollection", "customIntegrations"],
```

Comment:
We need some additional work in the `home` plugin so we can show the telemetry disclaimer in the Welcome page